rust/crd/src/lib.rs

```diff
@@ -487,11 +487,11 @@ impl AirflowConfig {
         AirflowConfigFragment {
             resources: ResourcesFragment {
                 cpu: CpuLimitsFragment {
-                    min: Some(Quantity("200m".to_owned())),
-                    max: Some(Quantity("4".to_owned())),
+                    min: Some(Quantity("100m".to_owned())),
```

Comment:
Attention: We might need different defaults for different roles (scheduler, worker etc.)